pkg/server/server.go

```diff
@@ -90,16 +95,16 @@ func (s *Server) removeNotifierAndDrain(l *getEventsListener) {
 	}
 }
 func (s *Server) GetEvents(request *tetragon.GetEventsRequest, server tetragon.FineGuidanceSensors_GetEventsServer) error {
-	return s.GetEventsWG(request, server, nil)
+	return fmt.Errorf("GetEvents not implemented")
```

Comment:
why is it not implemented?

Reply:
I don't think its ever used?

Reply:
But maybe to answer the question, I didn't have a correct ctx to add to it so I figured better to just make it bail out.

Reply:
hmm i think this is the grpc handler for GetEvents requests, so i'm guessing `getevents` command will break with this change: https://github.com/cilium/tetragon/blob/df691e27c5782041a935f3191a3a6067467d8304/cmd/tetra/getevents/getevents.go#L23

Reply:
Yeah this will break grpc :open_mouth:

Reply:
ah OK will fix.

Reply:
Fixed up well enough to not break existing stuff we might need a bit more to pass a closer through this API. But for now I'm going to leave it as is.